src/screens/CheckoutScreen.tsx

```diff
@@ -170,7 +181,7 @@ const CheckoutScreen = (props) => {
                                 value={contactInfoData[item.key] || ""}
                                 placeholder={item.placeholder}
                                 onPressIn={() => {
-                                    dispatch({ type: 'FILL_FIELDS', payload: 'dummydata' })}
+                                    dispatch({ type: 'FILL_FIELDS', payload: 'dummydata', onScope:email ? email:null})}
```

Comment:
@dgbailey this `onScope` stuff works great. Wondering, is this a property or part of the redux or React Context API? Or is it more like a property you're declaring and it simply gets passed through.